### DISCUSSION TITLE
wsl + jupyter notebook + debugging + output plots to interactive window

### DISCUSSION THREAD

**Discussion by pnwork** [microsoft/vscode-jupyter#12370] [Questions and Answers]
I use wsl + jupyter notebook + debugging cell. I want to output plots from debug console to interactive window, not to the original cell (as it is far away in the notebook, not convenient at all). How?????

For example, in Pycharm there is "plots" window where I can output plots. is it possible to recreate such workflow in vscode, i.e. is there a separate "Plots" output window? I guess it is interactive window? but how to connect this interactive window in the above settings?

**Comment by amunger** [comment]
If I understand what you're asking - we don't have a way to do that automatically, but you can send plots to the plot viewer window from the notebook's output.

You might also be interested in this feature request to enable the Interactive Window to connect to the same local kernel as a notebook https://github.com/microsoft/vscode-jupyter/issues/4573
